content/projects/first-personal-website/_index.md

```diff
@@ -16,6 +16,11 @@ tags:
 - html
 - css
 title: Build your first personal website
+learning_outcomes:
+- code_hosting
```

Comment:
I would change the namespace a bit for this one, it isn't eally pure code things. It's not like this project can be done in Python, Javascript or Java. It can only be done using very specifc tools:

rather 
```
- web_dev_github_pages
- web_dev_responsive_page_design
- web_dev_page_structure
- web_dev_visual_design_principles
```

The katas would be better with the `code` namespace.